conftest.py

```
@@ -0,0 +1,40 @@
+                "ENGINE": "django.db.backends.sqlite3",
+                "NAME": ":memory:",
+            }
+        },
+        INSTALLED_APPS=[
+            "django.contrib.auth",
+            "django.contrib.contenttypes",
+            "rest_framework",
+            "blockauth",
+        ],
+        ROOT_URLCONF="blockauth.urls",
+        SECRET_KEY="test-secret-key-not-for-production",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "SECRET_KEY"

(S106)